### DISCUSSION TITLE
Fixed Y-Axis Independent from values

### DISCUSSION THREAD

**Discussion by PlutoTuer** [patrykandpatrick/vico#293] [Questions] [answered]
Hi,
im trying to format the startAxis to be fixed to values 0 to 10. 
I am currently running into the problem that the maxValue of the StartAxis is the maximum value given into entryModelOf().
I however need the Y-Axis to remain static independent of the values in the graph.
How would you go on about doing that?

**Comment by Gowsky** [comment, marked as answer]
Hi @PlutoTuer, please see [`AxisValuesOverrider`](https://patrykandpatrick.com/vico/api/core/com.patrykandpatrick.vico.core.chart/-chart/#-1655685839%2FProperties%2F186187275).

**Reply by PlutoTuer** [reply]
Thanks!
`chart = lineChart(
                                axisValuesOverrider = AxisValuesOverrider.fixed(
                                    minY = 0f,
                                    maxY = 10f
                                )
                            ),`
